### ISSUE TITLE
Support for Chinese character sets

### ISSUE THREAD

**Issue by haitun3721** [openobserve/openobserve#4193]
### Description

Error when receiving Chinese character set:
openobserve: 2024-08-13T04:46:34.563898212+00:00 ERROR openobserve::handler::tcp_udp: Error while converting UDP message to UTF8 string: invalid utf-8 sequence of 1 bytes from index 409，
or
 openobserve: 2024-08-13T04:31:48.616726489+00:00 ERROR openobserve::handler::tcp_udp: Error while converting TCP message to UTF8 string: incomplete utf-8 byte sequence from index 1458,

### Proposed solution

Support for GBK character sets may need to be added

### Alternatives considered

或者其他方案

**Comment by haitun3721** [comment]
There are no detailed logs, and it is not known from which IP it was sent from, and it may be the GBK encoded logs

**Comment by hengfeiyang** [comment]
Yes, maybe it is GBK, but we only support UTF-8. i don't think we can support GBK.
